cdap-ui/app/logviewer/logviewer.less

```diff
@@ -82,5 +82,8 @@ body.state-logviewerhome {
       height: ~"-moz-calc(100vh - 166px)";
       height: ~"-webkit-calc(100vh - 166px)";
     }
+    &.program-logs .logs-table {
+      height: ~"calc(100vh - 160px)";
+    }
```

Comment:
is the specificity of this change in `cdap-ui/app/directives/log-viewer/log-viewer.less` not enough?

Reply:
Oh completely missed this comment. Yeah we apparently have css for logs table in logviewer directive and in the feature. Specifying the css for programs in logviewer directive was not sufficient.